core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/Runner.scala

```diff
@@ -391,8 +392,8 @@ object PodsConfig {
   implicit val volumeMountsConfReader: ValueReader[Volume.Mount] = ValueReader.relative { config =>
     Volume.Mount(
       name = config.getOrElse[String]("name", ""),
-      mountPath = config.getOrElse[String]("mountPath", ""),
-      readOnly = config.getOrElse[Boolean]("readOnly", false),
+      mountPath = config.getOrElse[String]("mount-path", ""),
```

Comment:
Should we start having a `Migrations.md` file to list user-facing breaking changes?

Reply:
FYI in this case, mounting volumes was introduced to mount secrets, which has not been released yet, so it's technically not a breaking change, since both will be released together. It's probably good enough to mention breaking changes in commit messages, and then we can add additional notes with the release.